src/index.ts

```diff
@@ -0,0 +1,8 @@
+function helloworld (msg = 'Hello World!'): string {
+  console.log(msg)
+  return msg
+}
+
+helloworld()
+
+export default helloworld
```

Comment:
Also take a look of the difference between `export default` and `export` and explain here, please.

Reply:
Both can be used to export constants, functions, files and modules but `Export` is used to export multiple objects while `export default` is used for a single export.